dpctl/tensor/_sorting.py

```diff
@@ -85,6 +85,8 @@ def sort(x, /, *, axis=-1, descending=False, stable=True, kind=None):
         return dpt.copy(x, order="C")
     else:
         axis = normalize_axis_index(axis, ndim=nd, msg_prefix="axis")
+    if x.size == 1:
```

Comment:
It seems already covered by #2106. Do we need to have the fix on python side as well?

Reply:
No, it's a temporary fix to keep coverage builds from failing, when #2106 is merged I'm going to drop the commit